src/cordova/apple/xcode/ios/Outline.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oh, I think you opened Outline.xcodeproj to make the edits. You'll need to revert them and open Outline.xcworkspace instead, so it doesn't create a new `project.xcworkspace`.

Always open the workspace, not the project.